install_ntf.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2019 Intel Corporation
+
+ENABLE_NTF=${ENABLE_NTF:-""}
+BESS_COMMIT=${BESS_COMMIT:-master}
+NTF_COMMIT=${NTF_COMMIT:-master}
+
+SUDO=''
+[[ $EUID -ne 0 ]] && SUDO=sudo
+
+install_ntf() {
+  set -e
+
+  $SUDO apt install -y  \
+    autoconf \
+    automake \
+    clang-format \
+    doxygen \
+    libjansson-dev \
+    libjansson4 \
+    libtool
+
+  wget -qO cjose.zip https://github.com/cisco/cjose/archive/master.zip
+  unzip cjose.zip
+  pushd cjose-master
+  ./configure
+  make
+  make install
+  popd
+
+  wget -qO ntf.zip https://github.com/Network-Tokens/ntf/archive/${NTF_COMMIT}.zip
+  unzip ntf.zip
+  cp ntf-${NTF_COMMIT}/bessctl/conf/* bess-${BESS_COMMIT}/bessctl/conf/
+  cp ntf-${NTF_COMMIT}/extra-ntf.mk bess-${BESS_COMMIT}/core/
```

Comment:
@nevelis can you try if moving extra-ntf.mk to modules in the upstream works? we can get rid of this line

Reply:
Upstream only includes `extra*.mk` from BESS's core directory, and the top-level directory of each plugin.

This is a bit of a pain huh :) If the plugins had their code in "core/modules", we could just copy the contents as-is directly into the BESS build and it will work.  But we wouldn't have to do this if --plugin worked properly.

I'll have to leave this line in there at the moment but will make a second pass on this if I find some time this weekend to look at the --plugin issue.

Reply:
yup agree as above